recipes/behaviortree.cpp/all/conanfile.py

```diff
@@ -58,38 +67,49 @@ def config_options(self):
             del self.options.fPIC
 
     def configure(self):
+        if conan_version >= Version("2.0.0") and Version(self.version) < "4.0.1":
+            raise Exception("Conan >=v2.0.0 is not compatible with behaviortree.cpp < 4.0.1")
```

Comment:
What is the reason for this error? We have not made 2.0 required so it seems odd to have this in 🤔 
The Conan client also prints a message so it should not be require in each recipe